### DISCUSSION TITLE
0.16.2

### DISCUSSION THREAD

**Discussion by kpcyrd** [kpcyrd/rebuilderd#91] [General]
- Switch worker-debian Dockerfile to use [debrebuild.py](https://github.com/fepitre/debrebuild) by @fepitre

## Thanks

We'd like to thank @SantiagoTorres and @repi for their support on [github sponsors](https://github.com/sponsors/kpcyrd).

<hr /><em>This discussion was created from the release <a href='https://github.com/kpcyrd/rebuilderd/releases/tag/v0.16.2'>0.16.2</a>.</em>

**Comment by fepitre** [comment]
I will work on the packaging to make it easy to install.
